locksmith/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "predev": "NODE_ENV=development sequelize db:migrate",
-    "dev": "nodemon server.js",
+    "dev": "tsc-watch --onSuccess 'node build/server.js' --onFailure 'echo Beep! Compilation Failed'",
```

Comment:
This command will run `tsc` in response to any change of watched files. It does incremental builds, so this is quick.

When the build finishes, it will run the compiled server. Any time it rebuilds, it kills the last server process and starts a new one.

When it fails it prints a cute message.